pkg/controller/csi-driver-lvm/actuator.go

```diff
@@ -0,0 +1,788 @@
+	return nil
+}
+
+func (a *actuator) controllerObjects(namespace string) ([]client.Object, error) {
+
+	csidriverlvmServiceAccountController := &corev1.ServiceAccount{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "csi-driver-lvm-controller",
+			Namespace: namespace,
+		},
+	}
+
+	csidriverlvmClusterRoleController := &rbacv1.ClusterRole{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "csi-driver-lvm-controller",
+			Namespace: namespace,
+		},
+		Rules: []rbacv1.PolicyRule{
+			{
+				APIGroups: []string{""},
+				Resources: []string{"persistentvolumes"},
+				Verbs:     []string{"get", "list", "watch", "update", "patch", "create", "delete"},
+			},
+			{
+				APIGroups: []string{"storage.k8s.io"},
+				Resources: []string{"csinodes"},
+				Verbs:     []string{"get", "list", "watch"},
+			},
+			{
+				APIGroups: []string{"storage.k8s.io"},
+				Resources: []string{"volumeattachments"},
+				Verbs:     []string{"get", "list", "watch", "update", "patch"},
+			},
+			{
+				APIGroups: []string{""},
+				Resources: []string{"persistentvolumeclaims"},
+				Verbs:     []string{"get", "list", "watch", "update", "patch"},
+			},
+			{
+				APIGroups: []string{""},
+				Resources: []string{"persistentvolumeclaims/status"},
+				Verbs:     []string{"update", "patch"},
+			},
+			{
+				APIGroups: []string{"storage.k8s.io"},
+				Resources: []string{"storageclaess"},
+				Verbs:     []string{"get", "list", "watch"},
+			},
+			{
+				APIGroups: []string{""},
+				Resources: []string{"events"},
+				Verbs:     []string{"get", "list", "watch", "update", "patch", "create", "delete"},
+			},
+			{
+				APIGroups: []string{""},
+				Resources: []string{"nodes"},
+				Verbs:     []string{"get", "list", "watch"},
+			},
+			{
+				APIGroups: []string{"storage.k8s.io"},
+				Resources: []string{"volumeattachments/status"},
+				Verbs:     []string{"patch"},
+			},
+			{
+				APIGroups: []string{""},
+				Resources: []string{"pods"},
+				Verbs:     []string{"get", "list", "watch"},
+			},
+		},
+	}
+
+	csidriverlvmClusterRoleBindingController := &rbacv1.ClusterRoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "csi-driver-lvm-controller",
+			Namespace: namespace,
+		},
+		Subjects: []rbacv1.Subject{
+			{
+				Kind:      "ServiceAccount",
+				Name:      "csi-driver-lvm-controller",
+				Namespace: namespace,
+			},
+		},
+		RoleRef: rbacv1.RoleRef{
+			APIGroup: "rbac.authorization.k8s.io",
+			Kind:     "ClusterRole",
+			Name:     "csi-driver-lvm-controller",
+		},
+	}
+
+	csiAttacherImage, err := imagevector.ImageVector().FindImage("csi-attacher")
+	if err != nil {
+		return nil, fmt.Errorf("failed to find csi-attacher image: %w", err)
+	}
+
+	csiResizerImage, err := imagevector.ImageVector().FindImage("csi-resizer")
+	if err != nil {
+		return nil, fmt.Errorf("failed to find csi-resizer image: %w", err)
+	}
+
+	var hostPathType corev1.HostPathType = corev1.HostPathDirectoryOrCreate
+	var replicas = int32(1)
```

Comment:
I think using these inline with `ptr.To` from `"k8s.io/utils/ptr"` could be a bit nicer.